### PR TITLE
Draw aicp poses in director

### PR DESCRIPTION
**Pull request by benoit-robotics**
@mauricefallon 

This new module subscribes to /aicp/poses and display them.
You can display a pose as a frame or an arrow, you can hide the lines between poses, you also can change the properties of all the lines at the same time if you change the properties of the folder "aicp path/lines".

An /aicp/poses message contains all the poses "computed", I don't redraw the poses already received each time a new message arrives, so I made the assumption that these poses don't change over time. I have created another branch ( draw-aicp-poses ) where this assumption is not made. 

It works with:
https://github.com/ori-drs/director_drs/pull/92

### CODE REVIEW

File: src/python/director/pathdrawer.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import director.objectmodel as om
+import director.rosutils as rosutils
+from director import tfdrawer as tf_draw
+import director.applogic as app
+
+from nav_msgs.msg import Path
+
+
+class AicpPoseSource(om.ContainerItem):
```
Comment (mauricefallon) [inline src/python/director/pathdrawer.py]: this class shouldn't be called AicpPoseSource. It shouldn't mentioned Aicp - as it is general and reused

PathSource would be a better name.

**Comment by mauricefallon** [comment]
this looks very good. 

Currently its not possible to subscribe to two different sources like this:
>        self.pathdrawer = pathdrawer.AicpPoseSource("aicp poses", "/aicp/poses", self.tfDrawer) 
>        self.pathdrawer2 = pathdrawer.AicpPoseSource("odom poses", "/aicp/odom_poses", self.tfDrawer) 
They overwrite one another.

You might need aicp running to properly test. I can help with that.

**Comment by benoit-robotics** [comment]
> Currently its not possible to subscribe to two different sources like this:
> 
> > ```
> >    self.pathdrawer = pathdrawer.AicpPoseSource("aicp poses", "/aicp/poses", self.tfDrawer) 
> >    self.pathdrawer2 = pathdrawer.AicpPoseSource("odom poses", "/aicp/odom_poses", self.tfDrawer) 
> > ```
> 
> They overwrite one another. Can you fix that?
> 

I wasn't able to test it, but now each item has an unique name so it should work.

**Comment by benoit-robotics** [comment]
> Currently its not possible to subscribe to two different sources like this:
> 
> > ```
> >    self.pathdrawer = pathdrawer.AicpPoseSource("aicp poses", "/aicp/poses", self.tfDrawer) 
> >    self.pathdrawer2 = pathdrawer.AicpPoseSource("odom poses", "/aicp/odom_poses", self.tfDrawer) 
> > ```
> 
> They overwrite one another. Can you fix that?
> 

I have tested it, it works. Can I merge this PR ? I would prefer to work on the visualisation of Yiduo's work in another PR.

**Pull request merged**